indico/modules/events/templates/footer.html

```diff
@@ -45,7 +45,7 @@ <h4>{% trans %}Calendaring{% endtrans %}</h4>
             </div>
             <div class="social-site">
                 <a href="https://www.google.com/calendar/event?{{ google_calendar_params|urlencode }}" target="_blank">
-                    <img src="//www.google.com/calendar/images/ext/gc_button1.gif" alt="" border="0">
+                    <img src="{{ indico_config.IMAGES_BASE_URL }}/google_calendar.gif" alt="" border="0">
```

Comment:
Please use `url_for` using the `assets.image` endpoint. You find examples if you search for that endpoint name in existing templates.